fastlane_core/lib/fastlane_core/project.rb

```diff
@@ -303,5 +303,67 @@ def parsed_info
     def automated_scheme_selection?
       !!ENV["AUTOMATED_SCHEME_SELECTION"]
     end
+
+    def backticks(command, print: true, timeout: false, retries: 0)
```

Comment:
Oh actually I think you referring to `Project.run_command` which has timeout but can't retry. I'll refactor this.